docs/organization/organisation.md

```diff
@@ -0,0 +1,142 @@
+
+- [webankOnlineBanking](https://github.com/ADORSYS-GIS/webank-OnlineBanking): This repository  contains the backend services of the project, implemented using Spring Boot. This repository handles business logic, data persistence, and API endpoints
+
+- [webank-AppManager](https://github.com/ADORSYS-GIS/webank-AppManager): This repository contains a Progressive Web App (PWA) built with React and TypeScript. This PWA offers feature and functionalities for the Teller Admin who has the sole role of managing teller access to the system, thereby: - Onboarding Teller
+	1. App, issuing TellerID after verifying the authenticity of the teller.
+	2. Offboarding Teller-App,
+	3. Suspending Teller-App when teller is on leave.
+
+- [webank-OwnerApp](https://github.com/ADORSYS-GIS/webank-OwnerApp): This repository host the Owner Progresive Web App which is built with react and typescript. This app has the following functionalities
+	1. Onboarding Admin-App (resp. Manager-App), issuing AdminID (resp. ManagerID) after verifying the  authenticity of each staff member.
+	2. Offboarding them,
+	3. Suspending those Apps whenever required.
+
+- [webank-TellerApp](https://github.com/ADORSYS-GIS/webank-TellerApp): This contains a PWA for the CFI staff managing banking users. It has the following functionalities
+	1. Onboarding User-App: The core element of the User-App activation consists in:
+	a. Validating the identity of a user claiming access to one or many bank accounts.
+	b. Validating the legitimacy of the claimed access to bank accounts by looking up in the CBS.
+	c. Issue the BankID to be returned to the User-App.
+	2. 1st Level Support: A teller has a sound understanding of processes and procedures implemented by the onlinebanking environment and acts as the first level support instance to banking users.
+	3. Offboarding User-App: The teller is also in charge of initiating the revocation of a User-App on the request of the bankinguser, after checking legitimacy of the revocation request.
+- [webank-AdminApp](https://github.com/ADORSYS-GIS/webank-AdminApp)  This is just another PWA with the following functionalities  
+    1. Onboarding Teller-App, issuing TellerID after verifying the authenticity of the teller.
+    2. Offboarding Teller-App,
+    3. Suspending Teller-App when teller is on leave.
+**Branching Strategy:** 
+**Commit Guidelines:** Use commut conventions in nthe best way possible
+**Pull Requests:** Pull request must be reviewed by at least two team members before merging.
+
+## Communication Channels
+**Slack/Teams/Other:** [slack channel](https://app.slack.com/client/T02977GUH/C07KTC0ER4L?cdn_fallback=1): This is the main communication channel for this project.
+- microsoft teams: For online meetings and other interactions.
+- [Online Banking whatsapp group](https://chat.whatsapp.com/HGdeHvl2Gs5DE50RrqE01k): For local updates and gist that conern the project
+
```

Comment:
I think slack and microsoft teams already is enough in case we are to be contacted... no need adding the whatsapp group.. it's congested..

please modify the spelling of **conern** i think you meant **concern**